src/systems/variational_smoother_constraint.C

```diff
@@ -521,6 +521,47 @@ void VariationalSmootherConstraint::constrain_node_to_line(const Node & node,
     }
 }
 
+void VariationalSmootherConstraint::find_nodal_or_face_neighbors(
+    const MeshBase &mesh, const Node &node,
```

Comment:
Sorry to continue harping on this, but we put spaces both before and after the `&` and `*`, so here it would be:
```
const MeshBase & mesh, const Node & node,
```
In addition we usually do a hanging indent when there are multiple lines of parameters rather than indenting a fixed four spaces.

I think the clang-format checked in to the repo handles this automatically (?), but I've never used it.